cpp/src/parquet/metadata.h

```diff
@@ -118,6 +119,12 @@ struct PageEncodingStats {
   int32_t count;
 };
 
+/// \brief Public struct for location to page index in ColumnChunkMetaData.
+struct IndexLocation {
+  int64_t index_file_offset_bytes;
+  int32_t offset_index_length;
```

Comment:
I'm not sure why the complex naming, we could just have
```suggestion
  /// File offset of the given index, in bytes
  int64_t offset;
  /// Length of the given index, in bytes
  int32_t length;
```

Reply:
Fixed